{"name": "test_eval_2[3+5-8]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 35 == 30", "trace": "test_input = '3+5', expected = 8\n\n    @pytest.mark.parametrize('test_input,expected',[('3+5',8),\n    ('2-1',1),('7*5',30)])\n    def test_eval_2(test_input,expected):\n        assert eval(test_input)==expected\n>       assert 35 == 30                     # 提示把30改成35\nE       assert 35 == 30\n\ntest_demo.py:15: AssertionError"}, "parameters": [{"name": "test_input", "value": "'3+5'"}, {"name": "expected", "value": "8"}], "start": 1663130405849, "stop": 1663130405849, "uuid": "f9f8d814-10ed-4b1c-a6d5-4f90328168a4", "historyId": "dfc4b9e567ceae71675d4f617f0aad26", "testCaseId": "4a96b9bee4b19b1cdeac3c287387c9ed", "fullName": "test_case.demo_case.stu_demo.test_demo#test_eval_2", "labels": [{"name": "parentSuite", "value": "test_case.demo_case.stu_demo"}, {"name": "suite", "value": "test_demo"}, {"name": "host", "value": "我不帅"}, {"name": "thread", "value": "7132-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.demo_case.stu_demo.test_demo"}]}